prog/install_rhizome.rb

```diff
@@ -22,12 +24,17 @@ class Prog::InstallRhizome < Prog::Base
               IO.copy_stream(_1, tf)
             end
           end
+          file_hash_map[file] = Digest::SHA384.file(full_path).hexdigest
         else
           # :nocov:
           fail "BUG"
           # :nocov:
         end
       end
+
+      writer.add_file("hashes.json", Integer("100755", 8)) do |tf|
```

Comment:
```suggestion
      writer.add_file("hashes.json", 0100755) do |tf|
```

Seems best to avoid unnecessary string to integer conversion.  If the `0` prefix is too obscure, you could use `0o100755`

Reply:
TIL `0o100755 == 33261` ℹ️